EIPS/eip-draft_permission_token.md

```diff
@@ -0,0 +1,317 @@
+
+### permissionRequire
+
+Return `true` if `_required` permission is a subset of `_permission` permission otherwise return `false`.
+
+```solidity
+function permissionRequire(uint256 _required, uint256 _permission) external pure returns (bool isPermission);
+```
+
+#### transfer
+
+Transfers a subset of `_permission` permission to address `_to`, and MUST emit the `Transfer` event.
+The function SHOULD `revert` if the message caller's account permission does not have the subset of the transferring permission. The function SHOULD `revert` if any of transferring permission is existing on target `_to` address.
+
+_Note_ Transfers of `0` permission MUST be treated as normal transfers and emit the `Transfer` event.
+
+```solidity
+function transfer(address _to, uint256 _permission) external returns (bool success)
+```
+
+#### approve
+
+Allows `_delegatee` to act for the permission owner's behalf, up to the `_permission` permission. If this function is called again it overwrites the current granted with `_permission`.
+
+**Note**:
+
+- `_permission` MUST be a subset of all available permission of permission owner.
+- `approve()` method SHOULD `revert` if granting `_permission` permission is not a subset of all available permission of permission owner.
+
+```solidity
+function approve(address _delegatee, uint256 _permission) external returns (bool success)
+```
+
+#### delegated
+
+Returns the subset permission of the `_owner` address were granted to `_delegatee` address.
+
+```solidity
+function delegated(address _owner, address _delegatee) external view returns (uint256 permission)
+```
+
+### Events
+
+#### Transfer
+
+MUST trigger when permission are transferred, including zero permission transfers.
+
+A token contract which creates new tokens SHOULD emit a `Transfer` event with the `_from` address set to `address(0x00)` when tokens are created.
+
+```solidity
+event Transfer(address indexed _from, address indexed _to, uint256 _value)
+```
+
+#### Approval
+
+MUST trigger on any successful call to `approve(address _delegatee, uint256 _permission)`.
+
+```solidity
+event Approval(address indexed _owner, address indexed _delegatee, uint256 _permission)
+```
+
+## Rationale
+
+- Permission tokens describe an alternative to Access Control Lists (ACLS) for providing authorisation and improving security.
+
+- Permission token balance of the relevant account in the supplied ecosystem is used to reflect permission and role.
+
+- The ecosystem operator will benefit from centrally managing all rights and responsibilities since it will make granting and revoking permissions easier.
+
+- The permission owner may assign a portion of their authority to the delegatee so that they may act on the smart contract in their place. This type of delegation is still in use today, as may be seen in daily life.
+
+- Cross-interactivity between different ecosystems will be simpler since permission tokens make it simple to issue authorization to external smart contracts.
```

Comment:
This section reads like motivation. Please move this content to the motivation section.

It is okay to have a placeholder for the rationale section.
```suggestion
## Rationale

Needs discussion.
```